osu.Game.Rulesets.Osu/Objects/Drawables/Pieces/SliderBall.cs

```diff
@@ -214,5 +173,68 @@ public void UpdateProgress(double completionProgress)
         {
             Position = slider.CurvePositionAt(completionProgress);
         }
+
+        private class FollowCircleContainer : Container
+        {
+            public override bool HandlePositionalInput => true;
+        }
+
+        public class DefaultFollowCircle : CompositeDrawable
+        {
+            public DefaultFollowCircle()
+            {
+                RelativeSizeAxes = Axes.Both;
+
+                InternalChild = new CircularContainer
+                {
+                    RelativeSizeAxes = Axes.Both,
+                    Masking = true,
+                    BorderThickness = 5,
+                    BorderColour = Color4.Orange,
+                    Blending = BlendingMode.Additive,
+                    Child = new Box
+                    {
+                        Colour = Color4.Orange,
+                        RelativeSizeAxes = Axes.Both,
+                        Alpha = 0.2f,
+                    }
+                };
+            }
+        }
+
+        public class DefaultSliderBall : CompositeDrawable
+        {
+            private readonly Bindable<Color4> accentColour = new Bindable<Color4>();
+
+            private Drawable drawableBall;
+
+            [BackgroundDependencyLoader]
+            private void load(DrawableHitObject drawableObject, ISkinSource skin)
+            {
+                RelativeSizeAxes = Axes.Both;
+
+                float radius = skin.GetValue<SkinConfiguration, float?>(s => s.SliderPathRadius) ?? OsuHitObject.OBJECT_RADIUS;
+
+                InternalChild = new CircularContainer
+                {
+                    Masking = true,
+                    RelativeSizeAxes = Axes.Both,
+                    Scale = new Vector2(radius / OsuHitObject.OBJECT_RADIUS),
+                    Anchor = Anchor.Centre,
+                    Origin = Anchor.Centre,
+                    BorderThickness = 10,
+                    BorderColour = Color4.White,
+                    Alpha = 1,
+                    Child = drawableBall = new Box
+                    {
+                        RelativeSizeAxes = Axes.Both,
+                        Alpha = 0.4f,
+                    }
+                };
+
+                accentColour.BindTo(drawableObject.AccentColour);
```

Comment:
It's not supposed to be the same colour as the hitobject, see https://github.com/ppy/osu/pull/5466/files#diff-d96c6b498fa4f267abf78a6213c2f609L169 . Also this needs to be set for the skin's slider ball also.